scripts/xtensa-build-zephyr.py

```diff
@@ -186,6 +187,16 @@ def parse_args():
 sign must be used (https://bugs.python.org/issue9334)""",
 	)
 
+	parser.add_argument("--key_name", required=False,
```

Comment:
@marc-hb Thank you for the suggestions! I will change the option to --key-type and remove `required=False`

Reply:
> `--key-type-subdirs`? It's still quite short and captures it all.

@marc-hb Do you mean `--key-type-subdir`?  We only specify one key type, not multiple key types, in a building.
To avoid ambiguous `--key-type-subdir ""` for the empty key type, we may define `choices=["community", "dbgkey", "none"]` and let user input `--key-type-subdir none` for the empty type.

Reply:
> Do you mean --key-type-subdir? We only specify one key type, not multiple key types, in a [single] build [per script invocation]

Good point, agreed.